azerothcore/azerothcore-wotlk
# fix(Core/DB): Unrestrict the amount of mails loaded

Partially reverts https://github.com/azerothcore/azerothcore-wotlk/commit/d5bb6ba1c46141f0fba5b130f9360d4df6ff712a



## Changes Proposed:
-  Remove the limiter added in https://github.com/azerothcore/azerothcore-wotlk/commit/d5bb6ba1c46141f0fba5b130f9360d4df6ff712a
-  It causes issues when loading mails, leading them to not be displayed in-game once mailbox space is freed.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/7564

## SOURCE:


## Tests Performed:

- Tested in-game.
- 


## How to Test the Changes:


1. Spam .send mail from console to a character that is offline at least 60-70 times
2. Check your full mail box
3. Delete some mails, notice new ones show up as soon as you reopen your mailbox

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
